<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="4" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-07-02T22:17:15 IST" name="Suite" finished-at="2023-07-02T22:18:13 IST" duration-ms="58327">
    <groups>
    </groups>
    <test started-at="2023-07-02T22:17:15 IST" name="Test" finished-at="2023-07-02T22:18:13 IST" duration-ms="58327">
      <class name="Test_Classes.Patch_TC1">
        <test-method signature="extractor()[pri:0, instance:Test_Classes.Patch_TC1@345f69f3]" started-at="2023-07-02T22:17:49 IST" name="extractor" finished-at="2023-07-02T22:18:06 IST" duration-ms="17093" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extractor -->
      </class> <!-- Test_Classes.Patch_TC1 -->
      <class name="Test_Classes.Post_TC1">
        <test-method signature="extractor()[pri:0, instance:Test_Classes.Post_TC1@20ca951f]" started-at="2023-07-02T22:17:15 IST" name="extractor" finished-at="2023-07-02T22:17:38 IST" duration-ms="22614" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extractor -->
      </class> <!-- Test_Classes.Post_TC1 -->
      <class name="Test_Classes.Put_TC1">
        <test-method signature="extractor()[pri:0, instance:Test_Classes.Put_TC1@457c9034]" started-at="2023-07-02T22:17:38 IST" name="extractor" finished-at="2023-07-02T22:17:48 IST" duration-ms="10686" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extractor -->
      </class> <!-- Test_Classes.Put_TC1 -->
      <class name="Test_Classes.Post_Retry_TC">
        <test-method signature="validator(java.lang.String,java.lang.String)[pri:1, instance:Test_Classes.Post_Retry_TC@50de186c]" started-at="2023-07-02T22:18:06 IST" name="validator" finished-at="2023-07-02T22:18:06 IST" duration-ms="0" status="FAIL">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Cannot inject @Test annotated Method [validator] with [class java.lang.String, class java.lang.String].
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Cannot inject @Test annotated Method [validator] with [class java.lang.String, class java.lang.String].
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection
at org.testng.internal.Parameters.checkParameterTypes(Parameters.java:436)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:350)
at org.testng.internal.Parameters.createParameters(Parameters.java:693)
at org.testng.internal.Parameters.handleParameters(Parameters.java:879)
at org.testng.internal.Parameters.handleParameters(Parameters.java:728)
at org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)
at org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:924)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validator -->
        <test-method signature="extractor()[pri:2, instance:Test_Classes.Post_Retry_TC@50de186c]" started-at="2023-07-02T22:18:06 IST" name="extractor" finished-at="2023-07-02T22:18:13 IST" duration-ms="7726" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extractor -->
      </class> <!-- Test_Classes.Post_Retry_TC -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
